app/models/user_auth_token.rb

```diff
@@ -57,8 +57,13 @@ def self.is_suspicious(user_id, user_ip)
     ips.uniq!
     return false if ips.empty? # first login is never suspicious
 
-    user_location = login_location(user_ip)
-    ips.none? { |ip| user_location == login_location(ip) }
+    if user_location = login_location(user_ip)
```

Comment:
if there are no locations shouldn't everything be suspicious?

Reply:
In that case, absence of MaxMindDb would cause every login to be suspicious. Is this something we want?